tests/cmds/test_trustedactivities.py

```diff
@@ -0,0 +1,386 @@
+    assert INVALID_CHARACTER_ERROR in result.output
+
+
+def test_update_when_duplicate_value_conflict_py42_raises_exception_prints_error(
+    runner, cli_state, trusted_activity_conflict_error
+):
+    cli_state.sdk.trustedactivities.update.side_effect = trusted_activity_conflict_error
+    command = [
+        "trusted-activities",
+        "update",
+        f"{TEST_RESOURCE_ID}",
+        "--value",
+        "test-case",
+    ]
+    result = runner.invoke(cli, command, obj=cli_state)
+    assert result.exit_code == 1
+    assert CONFLICT_ERROR.format("test-case") in result.output
+
+
+def test_update_when_description_limit_exceeded_py42_raises_exception_prints_error(
+    runner, cli_state, trusted_activity_description_limit_exceeded_error
+):
+    cli_state.sdk.trustedactivities.update.side_effect = (
+        trusted_activity_description_limit_exceeded_error
+    )
+    command = [
+        "trusted-activities",
+        "update",
+        f"{TEST_RESOURCE_ID}",
+        "--description",
+        ">250 characters",
+    ]
+    result = runner.invoke(cli, command, obj=cli_state)
+    assert result.exit_code == 1
+    assert DESCRIPTION_LIMIT_ERROR in result.output
+
+
+def test_remove_calls_delete_with_expected_params(runner, cli_state):
+    command = ["trusted-activities", "remove", f"{TEST_RESOURCE_ID}"]
+    runner.invoke(cli, command, obj=cli_state)
+    cli_state.sdk.trustedactivities.delete.assert_called_once_with(TEST_RESOURCE_ID)
+
+
+def test_remove_when_missing_resource_id_prints_error(runner, cli_state):
+    command = ["trusted-activities", "remove"]
+    result = runner.invoke(cli, command, obj=cli_state)
+    assert result.exit_code == 2
+    assert MISSING_RESOURCE_ID_ARG in result.output
+
+
+def test_remove_when_resource_id_not_found_py42_raises_exception_prints_error(
+    runner, cli_state, trusted_activity_resource_id_not_found_error
+):
+    cli_state.sdk.trustedactivities.delete.side_effect = (
+        trusted_activity_resource_id_not_found_error
+    )
+    command = ["trusted-activities", "remove", f"{TEST_RESOURCE_ID}"]
+    result = runner.invoke(cli, command, obj=cli_state)
+    assert result.exit_code == 1
+    assert RESOURCE_ID_NOT_FOUND_ERROR.format(TEST_RESOURCE_ID) in result.output
+
+
+def test_list_calls_get_all_with_expected_params(runner, cli_state):
+    command = ["trusted-activities", "list"]
+    runner.invoke(cli, command, obj=cli_state)
+    assert cli_state.sdk.trustedactivities.get_all.call_count == 1
+
+
+def test_list_with_optional_fields_called_get_all_with_expected_params(
+    runner, cli_state
+):
+    command = ["trusted-activities", "list", "--type", "DOMAIN"]
+    runner.invoke(cli, command, obj=cli_state)
+    cli_state.sdk.trustedactivities.get_all.assert_called_once_with(type="DOMAIN")
+
+
+def test_list_prints_expected_data(runner, cli_state, py42_response):
+    py42_response.data = json.loads(ALL_TRUSTED_ACTIVITIES)
+
+    def gen():
+        yield py42_response.data
```

Comment:
This is where using `create_mock_response` is beneficial, since it's a "real" `Py42Response` just with mock `requests.Response`, so the lookups get passed to `.data` normally.